spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { described_class.new }
+  let(:dish) { double :dish }
+  let(:price) { double :price }
+
+    describe '#add_menu_item' do
+      it 'adds a new item to the menu list' do
+        expect { menu.add_menu_item(dish, price) }.to change{menu.list.count}.by(1)
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)